sdk/common/src/main/scala/org/typelevel/otel4s/sdk/TelemetryResource.scala

```diff
@@ -94,13 +93,29 @@ sealed trait TelemetryResource {
 }
 
 object TelemetryResource {
+  private[otel4s] object ResourceAttributes {
+    val TelemetrySdkName: AttributeKey[String] =
+      AttributeKey("telemetry.sdk.name")
```

Comment:
Since we shouldn't depend on the experimental artifacts, the values are hardcoded instead.

Reply:
my main concern with this is that it will quietly change and we won't remember to keep up

Reply:
If I understand the strategy correctly, the promotion to the 'stable' convention should be the following:

1) keys under `TelemetryExperimentalAttributes` will be deprecated
2) `TelemetryAttributes` will be generated under the stable module

Hopefully, we will notice this and update the hardcoded attributes accordingly.